### ISSUE TITLE
dct:publisher pour indiquer l'origine de la notice

### ISSUE THREAD

**Issue by delahousse** [DOREMUS-ANR/doremus-ontology#23]
Il semble que `dct:publisher` avec une valeur bnf soit utilisé pour indiquer l'origine des notices
voir http://data.doremus.org/work/478c732c-6e96-3347-b7da-885a98af0050 ou 
http://data.doremus.org/publication/d85fd2e1-d36c-3395-9edb-24934f629396 

Je pense que ce n'est pas la meilleur propriété pour indiquer l'origine de la notice. On a plutot l'impression que la BnF est l'éditeur de la partition.

Je propose d'utiliser soit `dct:source` ou une propriété de l'ontology [prov-o]( https://www.w3.org/TR/prov-o/)  . 
En utilisant prov on pourrait avoir une tracabilité plus détaillé quand on aura besoin, par exemple sur un alignement ou une inférence.

<img width="718" alt="capture d ecran 2017-03-14 a 14 40 00" src="https://cloud.githubusercontent.com/assets/4135291/23903652/8878e36e-08c5-11e7-9c92-681121d0abbd.png">

**Comment by rtroncy** [comment]
I originally was the one who suggested to @pasqLisena to use the ```dct:publisher``` property. The definition reads: *An entity responsible for making the resource available.* and I assume that all our URIs refer to metadata about real world objects thus the BnF is the entity responsible for those metadata. 

I see though the potential of confusion and I don't want to enter a new http-range-14 debate but I don't think that ```dct:source``` adds any clarity (*A related resource from which the described resource is derived*).

What pro-o property do you have in mind?

**Comment by delahousse** [comment]
Hello,

For dct:publisher I understand the definition but I think in the spirit of
Dublin Core and in the mind of librarian, this apply to the resource
described (cd, book, article...) and not to the catalog record describing
the resource.
If we use prov-o we could use the property wasAttributedTo linking the
Entity (the notice) and the producer of the notice.
We even could use wasDerivedFrom to relate the rdf resource to the original
marc notice from were it was extracted.

Bonne soirée
Jean


​


https://www.w3.org/TR/prov-o/diagrams/starting-points.svg

Jean Delahousse

Independent Consultant
Semantic technologies, Linked Open Data
https://fr.linkedin.com/in/jeandelahousse
delahousse.jean@gmail.com  <delahousse.jean@gmail.com>
mob +33 6 01 22 48 55
skype jean.delahousse

2017-03-14 17:47 GMT+01:00 Raphael Troncy <notifications@github.com>:

**Comment by rtroncy** [comment]
The MARC notice does not have a URI we can re-use.

I don't see how the use of prov-o does resolve the issue. We are deliberately fuzzy when we mint new URIs to describe entities whether they identify a work / expression or a description of a work / expression (what you call the notice). In this respect, both ```dct:publisher``` or ```prov:wasAttributedTo``` will have the same effect.

**Comment by delahousse** [comment]
Bon, je ne suis pas sur que cet échange mène quelque part, mais je
maintiens que la propriété dct:publisher est conçue pour désigner l'éditeur
d'un article ou d'un livre ou d'un disque, mais pas l'éditeur de la notice
qui décrit cette oeuvre, et que l'ontologie prov-o est faite pour assurer
la traçabilité des données elle même, donc par exemple la traçabilité d'une
notice.

Mais c'est une suggestion et je me rangerai sans problème à l'avis de tous.

Jean

Jean Delahousse

Independent Consultant
Semantic technologies, Linked Open Data
https://fr.linkedin.com/in/jeandelahousse
delahousse.jean@gmail.com  <delahousse.jean@gmail.com>
mob +33 6 01 22 48 55
skype jean.delahousse

2017-03-14 23:36 GMT+01:00 Raphael Troncy <notifications@github.com>:

**Comment by pasqLisena** [comment]
I am seeing that `dct:publisher` can produce confusion.
But on the other side, also _prov-o_ has the same appications of the `dc` properties.

See slide 31 of this [tutorial](https://www.slideshare.net/dgarijo/provo-tutorial-dc2013-conference):

<img width="623" alt="schermata 2017-03-17 alle 12 02 38" src="https://cloud.githubusercontent.com/assets/4126222/24040554/fd238f84-0b09-11e7-8719-7f58ea1894a5.png">

**Comment by pierrechoffe** [comment]
If I may give my point of view, I see various questions here:

1. So far, the record (la notice) is not modeled as such, the Complex Work bears the record identifier and thus acts more or less as an "implied record",
2. but the actual record contains information dispatched on various classes, e.g. Event (F28) and Expression (F22)
3. using dct:publisher (or dct:creator for that matter) on either of these classes is therefore confusing
4. All the more so when it comes to describing a publication

Prov-o is a dynamic model and the example above is very similar with our own descriptions of "Descriptive Expression Assignment" ([see schema](https://drive.google.com/open?id=0B4s5dBwAmhJsdzFCelJBRy1sOFE)) and other "Assignments".

Actually, the M45 Descriptive Expression Assignment schema already contains the information we need to describe what we now call "publisher", (agent, date,...) and we could add further info if needed, like "date modified" etc. Would this be a solution ?

**Comment by rtroncy** [comment]
I'm not sure I see where there is confusion. The use case is pretty simple: data is already grouped into named graphs that identify the source organization publishing this data (e.g. BnF, Philharmonie). We want a simple way of identifying this information into a single triple pattern to avoid querying for each statement in which named graph it belongs to. This is the role, so far, of the ```dct:publisher``` property that we attach to top level entity in the DOREMUS model (see also the [URI construction rules](https://github.com/DOREMUS-ANR/marc2rdf/blob/master/URI.patterns.md))

**Comment by delahousse** [comment]
Dear Raphael,

Why don't you want to consider the use of prov-o which purpose is to trace
the provenance and transformation of data, with a clear frontier between
the data and metadata on the data ? It is now a very commun choice : MCC,
Office de publications, Unicancer...

Jean

Jean Delahousse

Independent Consultant
Semantic technologies, Linked Open Data
https://fr.linkedin.com/in/jeandelahousse
delahousse.jean@gmail.com  <delahousse.jean@gmail.com>
mob +33 6 01 22 48 55
skype jean.delahousse

2017-03-19 23:04 GMT+01:00 Raphael Troncy <notifications@github.com>:

**Comment by rtroncy** [comment]
I like PROV and I don't mind using it. I just don't see how this solves our specific problem. Let me rephrase your concern to make sure I understand it: you're arguing that stating a ```<:Work> dct:publisher <:BnF>``` is ambiguous since one does not know if BnF has published the Work or the description of the Work (record), correct? If we replace ```dct:publisher``` by ```prov:wasAttributedTo``` we have the same confusion. If we go via a more complex modeling as suggested by @pierrechoffe we will indeed create a new object that encapsulate the notion of record but this will make every single query more complex and thus null-ify the idea of not going through the named graph route.

Perhaps I'm missing something, but then, please, provide a full concrete example of your solution.

**Comment by delahousse** [comment]
je viens de parler à Raphael, je pense que l'on est en phase, je vais faire
un exemple.
bonne journée
jean

Jean Delahousse

Independent Consultant
Semantic technologies, Linked Open Data
https://fr.linkedin.com/in/jeandelahousse
delahousse.jean@gmail.com  <delahousse.jean@gmail.com>
mob +33 6 01 22 48 55
skype jean.delahousse

2017-03-20 8:53 GMT+01:00 Raphael Troncy <notifications@github.com>:

**Comment by FLERESCHE** [comment]
Je ne comprends pas bien pourquoi Raphaël a écrit "The MARC notice does not have a URI we can re-use. "
Les notices en InterXmarx comportent un attribut qui contient l'identifiant pérenne (ARK) de la notice : 
<Attr Nom="NumeroNotice">37031036</Attr>
<Attr Nom="IDPerenne">ark:/12148/cb370310366</Attr>
L'URL utilisée pour citer la notice est construite à partir de cet identifiant ARK. Par exemple, pour la notice du TUM de la Symphonie fantastique de Berlioz dans le catalogue général de la BnF est : http://catalogue.bnf.fr/ark:/12148/cb13908482b

Sinon, je suis d'accord avec Jean, il vaut mieux utiliser PROV, car le Dublin Core est généralement utilisé pour décrire les ressources elles-mêmes, pas les métadonnées qui les décrivent. Dans les données de la BnF converties en DC  (notamment pour les entrepôts OAI, et aussi par conséquent dans Gallica, Publisher est utilisé pour l'éditeur du document décrit.

**Comment by rtroncy** [comment]
> Je ne comprends pas bien pourquoi Raphaël a écrit "The MARC notice does not have a URI we can re-use. " Les notices en InterXmarx comportent un attribut qui contient l'identifiant pérenne (ARK) de la notice :
37031036
ark:/12148/cb370310366
L'URL utilisée pour citer la notice est construite à partir de cet identifiant ARK. Par exemple, pour la notice du TUM de la Symphonie fantastique de Berlioz dans le catalogue général de la BnF est : http://catalogue.bnf.fr/ark:/12148/cb13908482b

Parce que j'avais oublié ce détail :-) C'est bien de me corriger. Ceci dit, ça ne marchera par les notices MARC de la Philharmonie ou encore moins avec les notices XML de Radio France qui n'ont au mieux que des identifiants internes dans une base de données, donc le problème reste entier.

> Sinon, je suis d'accord avec Jean, il vaut mieux utiliser PROV, car le Dublin Core est généralement utilisé pour décrire les ressources elles-mêmes, pas les métadonnées qui les décrivent. 

Très bien, en attente de l'exemple de Jean.

**Comment by delahousse** [comment]
Si vous êtes ok je peux faire un exemple qui indique pour chaque ressource
créée la notice d'ou cette information est dérivée (quand on a un
identifiant pour la notice) et l'institution à l'origine de l'information,
cela en utilisant l'ontologie prov-o.
jean

Jean Delahousse

Independent Consultant
Semantic technologies, Linked Open Data
https://fr.linkedin.com/in/jeandelahousse
delahousse.jean@gmail.com  <delahousse.jean@gmail.com>
mob +33 6 01 22 48 55
skype jean.delahousse

**Comment by rtroncy** [comment]
Oui, mais il faut aussi prévoir les cas où la notice n'a pas forcément un URI (pérenneà) comme identifiant, quitte à en construire un (i.e. ne pas oublier de traiter les cas de la Philharmonie et de Radio France)

**Comment by delahousse** [comment]
ok

Jean Delahousse

Independent Consultant
Semantic technologies, Linked Open Data
https://fr.linkedin.com/in/jeandelahousse
delahousse.jean@gmail.com  <delahousse.jean@gmail.com>
mob +33 6 01 22 48 55
skype jean.delahousse

**Comment by pasqLisena** [comment]
Also the Philharmonie should have an uri for each file -- see https://github.com/DOREMUS-ANR/marc2rdf/issues/43

**Comment by rtroncy** [comment]
@delahousse proposed a modeling in [those slides](https://docs.google.com/presentation/d/1bGkyQVmDDhqeTeNCi3iDn0Kn-dllIqpzoK7I_QaeIOg/edit#)

**Comment by pasqLisena** [comment]
closed in https://github.com/DOREMUS-ANR/marc2rdf/commit/4bb64399b444e1ea94ac2e2b1c76779a80c3b47c
